doc/whats_new.rst

```diff
@@ -86,6 +86,8 @@ API
 
     - To unify and extend the behavior of :func:`mne.comupute_raw_covariance` relative to :func:`mne.compute_covariance`, the default parameter ``tstep=0.2`` now discards any epochs at the end of the :class:`mne.io.Raw` instance that are not the full ``tstep`` duration. This will slighly change the computation of :func:`mne.compute_raw_covaraince`, but should only potentially have a big impact if the :class:`mne.io.Raw` instance is short relative to ``tstep`` and the last, too short (now discarded) epoch contained data inconsistent with the epochs that preceded it.
 
+    - The default `picks=None` in :func:`mne.io.raw.filter` nows picks eeg, meg and seeg channels, by `Jean-Remi King`_
```

Comment:
and it probably needs to be `mne.io.Raw.filter`, case sensitivity of the class name